### ISSUE TITLE
Any Update?

### ISSUE THREAD

**Issue by Nightreaver** [TheCherry/ark-server-manager#2]
Hallo

Wie siehts mit nem update aus?
Einige Zeilen sind nicht einmal zu ende geschrieben.

Machst du da grad noch was?

Check mal ein was du hast.

LG

**Comment by TheCherry** [comment]
Leider habe ich nicht wirklich viel mehr, da mein Serverstart anstand habe ich alles nur scirpt technisch mit hilfe des arkmanager tools umgesetzt. Habe derzeit 5 Scripts im screen am laufen.

Ich werde aber sobald ich die Performance issues auf unserem Server etwas behoben habe, das Projekt wieder aufnehmen.

**Comment by Nightreaver** [comment]
Ja, weil es den anschein macht das du die scripte schon irgend wie fertig gestellt hast und für euren server benutzt. Und ich habe halt den eindruck das du mitten in der Arbeit angecheckt hast.

Es ist halt schwer jetzt was dran zu machen wenn ich nicht genau weiß wo es hingehen soll.

Ich würde jetzt einfach mal versuchen ne struktur rein zu bringen...

aber lad doch mal die scripte alle hoch?

**Comment by TheCherry** [comment]
Ja ich muss das ganze noch Ordenlich aufbauen, die Scripte kann ich 
leider nicht hochladen. Da einiges davon später als Plugin nur auf 
unseren Server laufen wird (z.B. die Aufnahme der Spielerzeiten). 
Außerdem ist es teilweise sehr unschön gelöst ... Wie gesagt ich 
verwende derzeit "ark-server-tools" und meine Scripte rufen das einfach 
nur auf. os.system(...) etc

Am 06.11.2015 um 12:36 schrieb Nightreaver:
